src/bmiCalculator.ts

```diff
@@ -0,0 +1,21 @@
+export const calculateBmi = (height: number, weight: number): string => {
```

Comment:
I would separate this into 2 functions:

`calculateBmi` which does the actual calculation
and `classifyBmi` which takes that number and returns a value.

Another change I would make is return a string literal type instead of just a string:

```typescript
type BmiClass = 'underweight' | 'normal' | 'overweight' | 'obese'; 
```